src/services/user/autoFollowUserAnon.js

```diff
@@ -0,0 +1,49 @@
+const {successResponse} = require('../../utils');
+const {v4: uuidv4} = require('uuid');
+const {User, UserFollowUser} = require('../../databases/models');
+const UsersFunction = require('../../databases/functions/users');
+
+const autoFollowUserAnon = async (req, res) => {
+  try {
+    const userFollowUserList = await UserFollowUser.findAll({
+      where: {
+        is_anonymous: false
+      }
+    });
+
+    const userFollowerAnonData = [];
+    await Promise.all(
+      userFollowUserList.map(async (followUser) => {
+        const anonId = await UsersFunction.findAnonymousUserId(followUser.user_id_followed);
+
+        const followAnonUser = await UserFollowUser.findOne({
+          where: {
+            user_id_follower: followUser.user_id_follower,
+            user_id_followed: anonId,
+            is_anonymous: true
+          }
+        });
+
+        if (!followAnonUser) {
+          userFollowerAnonData.push({
+            follow_action_id: uuidv4(),
+            user_id_follower: followUser.user_id_follower,
+            user_id_followed: anonId,
+            is_anonymous: true
+          });
+        }
+      })
+    );
+
+    let returnUserFollowUser = await UserFollowUser.bulkCreate(userFollowerAnonData);
+
+    return successResponse(res, 'Success Follow Anon Users', returnUserFollowUser);
+  } catch (error) {
+    console.log(error);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Avoid using `console.log` for error logging in production code. It's better to use a dedicated logging library that provides more flexibility and control over log levels, formats, destinations, etc. Consider replacing `console.log(error)` with a proper logging statement using a logging library like Winston or Bunyan.

```diff
- console.log(error);
+ logger.error(error);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->